cron/createFDQNBlockList.py

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# Copyright (c) 2014 Mozilla Corporation
+
+import boto
+import boto.s3
+import logging
+import netaddr
+import random
+import re
+import sys
+from datetime import datetime
+from datetime import timedelta
+from configlib import getConfig, OptionParser
+from logging.handlers import SysLogHandler
+from pymongo import MongoClient
+
+import os
+sys.path.append(os.path.join(os.path.dirname(os.path.abspath(__file__)), '../lib'))
+from utilities.toUTC import toUTC
+
+
+logger = logging.getLogger(sys.argv[0])
+
+
+def loggerTimeStamp(self, record, datefmt=None):
+    return toUTC(datetime.now()).isoformat()
+
+def initLogger():
+    logger.level = logging.INFO
+    formatter = logging.Formatter(
+        '%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+    formatter.formatTime = loggerTimeStamp
+    if options.output == 'syslog':
+        logger.addHandler(
+            SysLogHandler(
+                address=(options.sysloghostname, options.syslogport)))
+    else:
+        sh = logging.StreamHandler(sys.stderr)
+        sh.setFormatter(formatter)
+        logger.addHandler(sh)
+
+def genMeteorID():
+    return('%024x' % random.randrange(16**24))
+
+def isFQDN(fqdn):
+    try:
+        # We could resolve FQDNs here, but that could tip our hand and it's
+        # possible us investigating could trigger other alerts.  As such,
+        # validation will consist of making sure we have a dot noted string
+        #with two or more tokens
+        # Positive Example: example.com (2 tokens, 'example' and 'com' == valid)
+        # Positive Example: foo.example.com (3 tokens, 'example' and 'com' == valid)
+        # Negative Example: com (1 token, 'com' == invalid)
```

Comment:
What about if we moved this function into lib, and then took the examples in the comments and made unit tests for them?